variables.tf

```diff
@@ -170,39 +170,9 @@ variable "rejected_patches" {
 
 variable "patch_baseline_approval_rules" {
   description = "A set of rules used to include patches in the baseline. Up to 10 approval rules can be specified. Each `approval_rule` block requires the fields documented below."
-  type = list(object({
```

Comment:
why not to add `approve_until_date` to the existing variable instead of changing the type completely? this is a breaking change for users already using this module

Reply:
Yes it is possible. sorry i will make change and update the PR